images/linux/scripts/installers/2004/basic.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
+################################################################################
+##  File:  basic.sh
```

Comment:
is this file any different than 18.04? Maybe we can share it?

Reply:
They are _slightly_ different, but yeah, let's combine them